src/workspaces/dashboard/WorkspaceNotifications.test.tsx

```diff
@@ -0,0 +1,157 @@
+import { act, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
Original unit tests was converted to TSX. I checked that it passed with the new code before doing modifications to it.